webapp/channels/src/sass/routes/_admin-console.scss

```
@@ -812,75 +853,60 @@
-
-            i {
-                color: color.adjust(variables.$primary-color, $lightness: 10%);
-            }
         }
 
         &--active {
-            background: functions.alpha-color(variables.$white, 0.1);
-            color: variables.$white;
+            padding-left: 8px;
+            background: rgba(var(--sys-sidebar-text-rgb), 0.12);
+            border-left: 2px solid var(--sys-sidebar-active-border);
```

Stylelint (17.14.0)

[error] Expected "border-left" to come before "background" (order/properties-order)

(order/properties-order)